src/lib/matchers.rs

```diff
@@ -678,9 +677,9 @@ mod tests {
                             "{\n\"src\": 1,\n\"dest\": 3\n}",
                             "{\n\"src\": 0,\n\"dest\": 0\n}",
                             "{\n\"src\": 2,\n\"dest\": 4\n}"]
-                .iter()
-                .map(|s| String::from(*s))
-                .collect::<Vec<String>>();
+            .iter()
```

Comment:
Admittedly, neither indentation level makes any sense to me whatsoever... If we're going to use rustfmt vanilla, we might need to split this into two logical lines, as at the moment it's just too confusing for a human to read IMHO.